Evolutionary_loop/Loop_Parts/Part_A/Part_A.sh

```diff
@@ -11,7 +11,7 @@
 #     07/02/23  Original version
 #
 #  Notes:
-# 		* vertical ruler at column 80
+# 		* vertical ruler at column 80; tab size 2
```

Comment:
Elsewhere we discussed using 4 spaces instead of hard tabs, since e.g. on GitHub they have size 8 by default, whereas 4 spaces are 4 spaces everywhere.

Reply:
Is there a way to swap out tabs with spaces? 
The auto-indentation seems to be inserting tabs as well.
Or perhaps do we want to change the Github tab display size to 4?

Reply:
You can do a find/replace to substitute tabs with spaces, and set your editor to enter 4 spaces when you press the `tab` key on your keyboard.
There is no way to change the display size on GitHub, also because I think it depends on the browser itself -- not on GitHub.

Reply:
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/managing-your-tab-size-rendering-preference

Is this not it?
Cuz on my GitHub account my tab display size is set to 2

Reply:
I wasn't aware GitHub had an option to override the default tab size, thanks for pointing that out!
However it doesn't quite solve the problem -- everyone will have to know about it and set it to `2` in order to see the code in those files indented properly, and the code in different files/repos that uses a different size will look off.
Spaces have the same size everywhere.